src/utils.ts

```diff
@@ -71,9 +71,14 @@ export const isWritableElement = (
 export const getFontFamilyString = ({
   fontFamily,
 }: {
-  fontFamily: FontFamily;
+  fontFamily: FONT_FAMILY;
 }) => {
-  return `${FONT_FAMILY[fontFamily]}, ${WINDOWS_EMOJI_FALLBACK_FONT}`;
+  for (const [fontFamilyString, id] of Object.entries(FONT_FAMILY)) {
+    if (id === fontFamily) {
+      return `${fontFamilyString}, ${WINDOWS_EMOJI_FALLBACK_FONT}`;
+    }
+  }
+  return WINDOWS_EMOJI_FALLBACK_FONT;
```

Comment:
I'm not sure what this is about. Is the looping through entries suppose to account for potentially incorrectly using fontFamily names instead of numbers?

The fallback should use both `Virgil, WINDOWS_EMOJI_FALLBACK_FONT` tho, no?

Reply:
In fact, unless you've used `enum` precisely for the above purpose (and even then), I'd use a regular object instead because enums transpile to contain the values in the keys as well:

```
{1: "Virgil", 2: "Helvetica", 3: "Cascadia", Virgil: 1, Helvetica: 2, Cascadia: 3}
```

Thus, consumers (both us and host apps) may get confused and think that the non-numeric values are allowed.

Reply:
yeah that's why I had changed the values to `string`, since we want to keep numbers I would also go for an object than enums

Reply:
To double back to this. Now that we've moved to an object, we should revert this to

```
return `${FONT_FAMILY[fontFamily]}, ${WINDOWS_EMOJI_FALLBACK_FONT}`;
```

Correct?

Reply:
But my point about the fallback stands? Should fall back to `Virgil, <Emoji>` no?

Reply:
I think we should have a reverse map of `Record<fontFamilyNumber, fontFamilyString>` for this.

Reply:
But if it's just one, two cases, then `findKey()` or something will work, too.

Reply:
I think it's ok to fall back to `WINDOWS_EMOJI_FALLBACK_FONT` for now as that gives a clear signal that font is not available instead of falling back to `Virgil` and this will only happen in case of package.